### DISCUSSION TITLE
tp.file.move copies the file instead of cutting it

### DISCUSSION THREAD

**Discussion by Archie-2021** [SilentVoid13/Templater#642] [Help] [answered]
Am I doing something wrong or it is supposed to be this way , in documentation it say it supposed to move the file meaning cut and pasting it

PS. I found that why this was happening was that it caused and error. If I run the script in the note rather than using templater menu . The code is on the file as it needs to move it so it causes issues, I need to auto execute it tho, I would appreciate if anyone can help me on how to do that?

**Comment by welpdx** [comment, marked as answer]
Make sure you have `await` in front of `tp.file.move`. 

If you still have an issue, see #318. 

If #318 isn't your issue, please elaborate on the steps that lead to your problem so I can understand what you are trying to do. Thanks.

Hope that helps

~welp

**Reply by Archie-2021** [reply]
> Make sure you have `await` in front of `tp.file.move`.
> 
> If you still have an issue, see #318.
> 
> If #318 isn't your issue, please elaborate on the steps that lead to your problem so I can understand what you are trying to do. Thanks.
> 
> Hope that helps
> 
> ~welp

Yes the issue was my rookie mistake of not using await. It work fine now.

Thank you for this awesome plugin, I am liking it more and more

**Reply by welpdx** [reply]
I happy to help. Please mark post as `Answered`. Thanks

> Thank you for this awesome plugin, I am liking it more and more

I like this plugin very much too. Credits to @SilentVoid13
